data/lib/src/datasource_impl/document_datasource_impl.dart

```diff
@@ -127,4 +127,60 @@ class DocumentDataSourceImpl implements DocumentDataSource {
 
     return DownloadTaskId(taskId);
   }
+
+  @override
+  Stream<Uri> downloadDocumentIOS(Document document, Token token, Uri baseUrl, CancelToken cancelToken) {
```

Comment:
We might need to track downloading progress in the future. So I decided to use `Stream` here